src/plugins/default-value-calculation/Findings.md

```diff
@@ -0,0 +1,131 @@
+# Default value calculation
+
+This Markdown file shortly describes my found requirements for default value calculation:
```

Comment:
From where did you get any information. How did you come to the conclusions?

Reply:
By looking through all of cassandras and lcdproc's configurations as you suggested.

This is more like a proposal like every other plugin PR we did.Especially the README.md is of interest. The Findings.md is just for backing up my suggested plugin solution and argue why I chose certain properties of the plugin.

Reply:
Btw. the problem is throughout the text, I only marked the first place.  For example it is unclear if the later section "Auto-Detection" now also describes requirements and if it is derived from the same applications.